main_COPY_AND_RENAME.py

```diff
@@ -29,8 +29,28 @@
         if event.type == pygame.QUIT:  # When user clicks the 'x' on the window, close our game
             pygame.quit()
             sys.exit()
-        
 
+    # Keyboard events
+    keys_pressed = pygame.key.get_pressed()
+    if keys_pressed[pygame.K_UP]:
+        # Replace this line
+    if keys_pressed[pygame.K_LEFT]:
+        # Replace this line
+    if keys_pressed[pygame.K_RIGHT]:
+        # Replace this line
+    if keys_pressed[pygame.K_DOWN]:
+        # Replace this line
+
+
+    # Mouse events
+    mouse_buttons = pygame.mouse.get_pressed()
+    # Runs while you hold it
+    if mouse_buttons[0]:  # If left mouse pressed
+        mouse_pos = pygame.mouse.get_pos()  # Get position of mouse
+        # Returns (x, y) coordinates as tuple
+
+    if mouse_buttons[2]:  # If right mouse pressed
+        # Replace this line
```

Comment:
1.  You can move `mouse_pos` above the entire `mouse_buttons` definition (because you can still use the mouse position without clicking). 
2. You can delete the comment `# Runs while you hold it` because it gives no value without your version 1 code
3. Also don't forget to add the `pass` keyword to your empty blocks
```suggestion
    # Mouse events
    mouse_pos = pygame.mouse.get_pos()  # Get position of mouse as a tuple representing the (x, y) coordinate
    mouse_buttons = pygame.mouse.get_pressed()
    if mouse_buttons[0]:  # If left mouse pressed
        pass  # Replace this line
    if mouse_buttons[2]:  # If right mouse pressed
        pass  # Replace this line
```